src/parser/jobs/mch/modules/Reassemble.tsx

```diff
@@ -1,86 +1,108 @@
-import {Trans, Plural} from '@lingui/react'
-import {ActionLink} from 'components/ui/DbLink'
-import {ActionRoot} from 'data/ACTIONS/root'
-import STATUSES from 'data/STATUSES'
-import {CastEvent, BuffEvent} from 'fflogs'
-import Module, {dependency} from 'parser/core/Module'
-import Combatants from 'parser/core/modules/Combatants'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
```

Comment:
Since you only ever use actions, could prob leave this as `ActionLink`. The new syntax works there too, so you can still `<ActionLink action="REASSEMBLE"/>` as needed.

Reply:
There's no need to do so, though. Why use individual ones when the data link works for everything.